src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  display: flex;
+  margin-right: 48px;
+  margin-bottom: 46px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.